src/main/java/com/android/volley/AsyncCache.java

```diff
@@ -24,5 +24,12 @@
      */
     public abstract void get(String key, OnGetCompleteCallback callback);
 
+    public interface OnPutCompleteCallback {
+        /** Invoked when the put to the cache is complete. */
+        void onPutComplete();
+    }
+
+    public abstract void put(String key, Cache.Entry entry, OnPutCompleteCallback callback);
```

Comment:
could you add Javadoc for this (like get)

Reply:
added javadoc here